nipype/workflows/smri/freesurfer/autorecon1.py

```diff
@@ -26,9 +28,11 @@ def checkT1s(T1_files, cw256=False):
     origvol_names = ["{0:03d}.mgz".format(i + 1) for i in range(len(T1_files))]
 
     # check if cw256 is set to crop the images if size is larger than 256
-    if not cw256 and any(dim > 256 for dim in shape):
+    if not cw256 and (voxel_size[0] * t1_size[0] > 256) or (voxel_size[1] * t1_size[1] > 256) or (voxel_size[2] * t1_size[2] > 256):
         print("Setting MRI Convert to crop images to 256 FOV")
         cw256 = True
+    else:
+        print("No need to add -cw256 flag")
```

Comment:
I have some data, which size of image is small than 256, but it still give me the error that we should use the cw256 flag, so I think the FOV is specified in mm, which corresponds to number of voxels multiply dimension of voxels. 
And for the nipype logging, I followed the site from nipype, and I know that in nipype, we can not use python logging module, which is a bug and you did not figure it out, and the nipyep logging [page()](http://nipype.readthedocs.io/en/latest/users/config_file.html) is not so clear to follow, so do you have some more elaborate page to follow???